### DISCUSSION TITLE
sendOtp function in GoTrueAdmin.ts

### DISCUSSION THREAD

**Discussion by jzambrano12** [supabase/supabase#16724] [Feature Requests]
I think it would be very useful to have a function called `sendOtp` to send an OTP code to an email or phone number that is not registered in the project.

Particularly in my current development I need to be able to link an email or phone number to a user who has lost access to my platform. I cannot do this because the `resend` function only works if the email or phone number you are trying to send the code to is registered in the project.

It would help a lot of people, I'm sure I'm not the only one that this happens to.

**Comment by VIRAT9358** [comment]
I believe we should have a recovery email registered apart from the main email. We can send OTP to that email ,it would make authentication process secure.

**Reply by jzambranoavvy** [reply]
In my case that's not a option. My users had a phone registered in their accounts and at the beginning it was enough but now we find it convenient that these users can link a different alternative to their accounts.

The problem is when these users do not correctly receive the otp codes to their phone number. In that case the idea is to help them linking an email to the account that matches their phone numbers.

In this particular case I see a need for the `sendOtp` function.
